### PR TITLE
Style Migration

### PR DESCRIPTION
**Pull request by alicesf2**
## Summary

Replaced bridge with semantic ui react, minor styling edits.

## Changes

- Login page button and alerts
- Profile page inputs, alerts, button
- Navbar styling

## Screenshots

Screen Shot 2021-01-25 at 6 18 11 PM (2)
Screen Shot 2021-01-25 at 6 18 17 PM (2)

Refs #40

### CODE REVIEW

**Review by mattwalo32 — APPROVED** [review]
All looks good, great work!

File: package.json [hunk 1]
```diff
@@ -1,45 +1,5 @@
...
-    "./**/*.{js,css,scss,json,md,jsx}": [
-      "prettier --write"
-    ],
-    "./api/**/*.{js,jsx}": [
-      "eslint --fix -c api/.eslintrc.json"
-    ],
-    "./client/**/*.{js,jsx}": [
-      "eslint --fix -c client/.eslintrc.json"
-    ]
+    "husky": "^4.3.8"
```
Comment (Qrtn) [inline package.json:3]: Did you accidentally delete a lot of the package.json?

**Review by Qrtn — CHANGES_REQUESTED** [review]
Looks great!

File: client/src/css/Login.css [hunk 2]
```diff
@@ -40,7 +39,7 @@
   margin: 2px 5px 2px 5px;
 }
 
-.login-btn:hover {
+/* .login-btn:hover {
```
[the remark this answers is not in the document]
Comment (alicesf2) [reply client/src/css/Login.css:42]: Ah yes my bad

File: package.json (hunk 1 above)
Comment (alicesf2) [reply package.json:3]: Yea I think I messed it up because husky was not working for me, will fix that

**Pull request merged**